cruise-control/opt/cruise-control/start.sh

```diff
@@ -1,4 +1,8 @@
 #!/bin/bash
 set -e
 cd /opt/cruise-control
+
+# Override heap settings for container environments
+export KAFKA_HEAP_OPTS="-XX:InitialRAMPercentage=30 -XX:MaxRAMPercentage=70 -XX:MinRAMPercentage=80"
```

Comment:
Needs to do like cruise-control-server-start.sh, set only if undefined. Otherwise if we merge it would override any env set at container start.